app/views/posts/show.html.erb

```diff
@@ -1 +1,17 @@
-<h1>Here is a list of posts given a user id and post id</h1>
+<h1>Welcome to <%= @post.author.name %>'s post</h1>
+
+<section class="container">
+    <div class="post">
+        <h3><%= @post.title %> by <%= @post.author.name %></h3>
+        <div class="counter">
+            <span>Comments: <%= @post.comments_counter %>, </span>
+            <span>Likes: <%= @post.likes_counter %> </span>
+        </div>
+        <p><%= @post.text %></p>
+    </div>
+    <div class="comment">
+        <% @post.recent_comments.each do |comment| %>
```

Comment:
- [ ] @Rhaegar121 Great work on your post-show page, however on this page, we are supposed to view all comments, but you have used the `recent_comments` method that only displays the 5 most recent comments.